src/equations/compressible_navier_stokes_2d.jl

```diff
@@ -113,167 +115,189 @@ struct GradientVariablesEntropy end
-  μ, Pr = promote(mu, Prandtl)
-
-  # Under the assumption of constant Prandtl number the thermal conductivity
-  # constant is kappa = gamma μ / ((gamma-1) Pr).
-  # Important note! Factor of μ is accounted for later in `flux`.
-  kappa = gamma * inv_gamma_minus_one / Pr
-
-  CompressibleNavierStokesDiffusion2D{typeof(gradient_variables), typeof(gamma), typeof(equations)}(gamma, inv_gamma_minus_one,
-                                                                                                    μ, Pr, kappa,
-                                                                                                    equations, gradient_variables)
+    gamma = equations.gamma
+    inv_gamma_minus_one = equations.inv_gamma_minus_one
+    μ, Pr = promote(mu, Prandtl)
+
+    # Under the assumption of constant Prandtl number the thermal conductivity
+    # constant is kappa = gamma μ / ((gamma-1) Pr).
+    # Important note! Factor of μ is accounted for later in `flux`.
+    kappa = gamma * inv_gamma_minus_one / Pr
+
+    CompressibleNavierStokesDiffusion2D{typeof(gradient_variables), typeof(gamma),
+                                        typeof(equations)}(gamma, inv_gamma_minus_one,
+                                                           μ, Pr, kappa,
+                                                           equations, gradient_variables)
 end
 
 # TODO: parabolic
 # This is the flexibility a user should have to select the different gradient variable types
 # varnames(::typeof(cons2prim)   , ::CompressibleNavierStokesDiffusion2D) = ("v1", "v2", "T")
 # varnames(::typeof(cons2entropy), ::CompressibleNavierStokesDiffusion2D) = ("w2", "w3", "w4")
 
-varnames(variable_mapping, equations_parabolic::CompressibleNavierStokesDiffusion2D) =
-  varnames(variable_mapping, equations_parabolic.equations_hyperbolic)
+function varnames(variable_mapping,
+                  equations_parabolic::CompressibleNavierStokesDiffusion2D)
+    varnames(variable_mapping, equations_parabolic.equations_hyperbolic)
+end
 
 # we specialize this function to compute gradients of primitive variables instead of
 # conservative variables.
-gradient_variable_transformation(::CompressibleNavierStokesDiffusion2D{GradientVariablesPrimitive}) = cons2prim
-gradient_variable_transformation(::CompressibleNavierStokesDiffusion2D{GradientVariablesEntropy}) = cons2entropy
-
+function gradient_variable_transformation(::CompressibleNavierStokesDiffusion2D{
+                                                                                GradientVariablesPrimitive
+                                                                                })
+    cons2prim
+end
+function gradient_variable_transformation(::CompressibleNavierStokesDiffusion2D{
+                                                                                GradientVariablesEntropy
+                                                                                })
+    cons2entropy
+end
```

Comment:
These seem a little awkward - any way to keep type parameters together, more like 
```julia
function foo(
       ::Type{Parameter}, ...
       ) = ...
``` 
instead?

Reply:
Feel free to try - but I don't think so, not with the current JuliaFormatter capabilities